x-pack/plugin/security/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -31,6 +31,9 @@ grant {
   // needed for kerberos file permission tests to access user information
   permission java.lang.RuntimePermission "accessUserInformation";
   permission java.lang.RuntimePermission "getFileStoreAttributes";
+
+  // TODO the ES security code also needs this permission (in addition to nimbus)
+  permission java.lang.RuntimePermission "accessDeclaredMembers";
```

Comment:
I think that you can simply remove the "TODO" here, leaving the remainder of the comment.

Reply:
Absolutely, the PR still needs some smaller touch ups. I have it here as a reference for the current thinking around where the `doPrivileged` calls would go